docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -1,10 +1,220 @@
 [[get-started-docker]]
 == Running the {stack} on Docker
 
-The Elastic Docker registry contains Docker images for all the products in the 
-{stack}. For more information, see:
+The Elastic Docker registry contains Docker images for all the products in 
+the {stack}: https://www.docker.elastic.co/
+
+[float]
+[[run-stack-docker]]
+=== Run with Docker Compose
+
+To get the default distributions of {es} and {kib} up and running in Docker, 
+you can use `docker-compose`:
+
+. Pull the docker images for {es} and {kib}:
++
+["source","sh",subs="attributes"]
+--------------------------------------------
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
+docker pull docker.elastic.co/kibana/kibana:{version}
+--------------------------------------------
+
+. Create a `docker-compose.yml` file for the Elastic Stack:
++
+["source","yaml"]
+--------------------------------------------
+include::docker/docker-compose.yml[]
+--------------------------------------------
+
+. Run `docker-compose` to bring up a three node Elasticsearch cluster and Kibana:
++
+["source","sh",subs="attributes"]
+--------------------------------------------
+VERSION={version} docker-compose up
+--------------------------------------------
+
+. Submit a `_cat/nodes` request to see that the nodes are up and running:
```

Comment:
We may want to link to the cat nodes API here.

It also may be confusing to include a console snippet *before* we explicitly tell people to open up Kibana.

Reply:
In the getting started content, I'm opting to minimize linking folks out of the flow.

Reply:
Makes sense. Thanks!